Sources/JWTKeychain/Routes/Api/ApiUserRoutes.swift

```diff
@@ -36,25 +36,27 @@ public struct UserRoutes: RouteCollection {
         jwtAuthMiddleware: AuthMiddleware? = nil,
         authMiddleware: Middleware = Auth.AuthMiddleware<User>(),
         protectMiddleware: ProtectMiddleware = ProtectMiddleware(
-            error: Abort.custom(
-                status: .unauthorized,
-                message: Status.unauthorized.reasonPhrase
-            )
+        error: Abort.custom(
+            status: .unauthorized,
+            message: Status.unauthorized.reasonPhrase
+        )
         ),
-        userController: UserControllerType? = nil
-    ) throws {
+        userController: UserControllerType? = nil,
+        mailer: MailerType
+        ) throws {
         self.drop = drop
         let config = try configuration ?? Configuration(drop: drop)
         self.configuration = config
         self.jwtAuthMiddleware = jwtAuthMiddleware ?? JWTKeychain.AuthMiddleware(configuration: config)
         self.authMiddleware = authMiddleware
         self.protectMiddleware = protectMiddleware
-        self.controller = userController ?? UserController(configuration: config, drop: drop)
+        self.mailer = mailer
```

Comment:
If you want, you could fallback to `Mailer` so simplify the callsite (as we do e.g. with `UserController`).